applications/nrf_desktop/src/events/passkey_event.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <stdio.h>
+
+#include "passkey_event.h"
+
+
+static int log_passkey_input_event(const struct event_header *eh, char *buf,
+				   size_t buf_len)
+{
+	const struct passkey_input_event *event = cast_passkey_input_event(eh);
+
+	return snprintf(buf, buf_len, "passkey: %" PRIu32, event->passkey);
+}
+
+static void profile_passkey_input_event(struct log_event_buf *buf,
+					const struct event_header *eh)
+{
+	const struct passkey_input_event *event = cast_passkey_input_event(eh);
+
+	ARG_UNUSED(event);
```

Comment:
This does not look correct

Reply:
It's the same below

Reply:
In case profiler is not enabled, the `profiler_log_encode_u32` is defined as empty static inline void function. I removed `ARG_UNUSED` locally and it caused no problem for me (no compiler warnings), so I will add this as separate commit.